site/docs/reference/Connectors/materialization-connectors/Firebolt.md

```diff
@@ -0,0 +1,83 @@
+# Firebolt
+
+This Flow connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of Flow collections into Firebolt `FACT` or `DIMENSION` tables.
+
+To interface between Flow and Firebolt, the connector creates an _external table_:
+a virtual table in an S3 bucket used as a staging area.
```

Comment:
I would reword this to be
```suggestion
a table connected to an S3 bucket, acting as a SQL interface to JSON documents stored in the S3 bucket. (See [Working with external tables](https://docs.firebolt.io/loading-data/working-with-external-tables.html))
```

Since the table itself is in Firebolt, but it acts as a SQL interface on over the JSON documents there.